rts/Sim/Units/Unit.cpp

```diff
@@ -328,14 +328,9 @@ void CUnit::PostInit(const CUnit* builder)
 	// does nothing for LUS, calls Create+SetMaxReloadTime for COB
 	script->Create();
 
-	// all units are blocking (ie. register on the blocking-map
-	// when not flying) except mines, since their position would
-	// be given away otherwise by the PF, etc.
-	// NOTE: this does mean that mines can be stacked indefinitely
-	// (an extra yardmap character would be needed to prevent this)
 	immobile = unitDef->IsImmobileUnit();
```

Comment:
Doesn't seem to need this line too.

Reply:
Not in this init function, but it sets the member which is used in a bunch of other places, e.g.
https://github.com/beyond-all-reason/spring/blob/f9c53c14af7cc3b237803a5267aab5c747f676fc/rts/Sim/Units/CommandAI/BuilderCAI.cpp#L1680-L1681